daemon/README.md

```diff
@@ -1,2 +1,48 @@
 # 40swapd
-The 40swap daemon (40swapd) interacts with the lightning node to manage the swap process. It features a cli and the daemon itself, the CLI can be used to manually invoke swaps and other configuration operations.
\ No newline at end of file
+The 40swap daemon (40swapd) interacts with the lightning node to manage the swap process. It features a cli and the daemon itself, the CLI can be used to manually invoke swaps and other configuration operations.
+
+## Configuration
+
+You can configure the 40swap daemon using environment variables or command line arguments. Check the section below or do `40swapd -h` for more information.
+
+## Docs (40swapd -h)
```

Comment:
I would remove this, it will be outdated very fast

Reply:
Indeed, removed